frontend/src/components/shared/richText.js

```diff
@@ -110,24 +114,28 @@ const formatNode = ({ type, format, level, image, children }) => {
           break;
       }
     case "paragraph":
+      return <p>{formatParagraph(children)}</p>;
     default: // Treat default case as regular paragraph
       return <div>{formatParagraph(children)}</div>;
   }
 };
 
 const RichText = ({ data, addPaddingToParagraphs = false }) => {
+  console.log(data);
   if (!Array.isArray(data)) {
     return "";
   }
 
   return (
-    <div>
-      {data.map((node, idx) => (
-        <div key={idx} className={addPaddingToParagraphs ? "my-3" : ""}>
-          {formatNode(node)}
-        </div>
-      ))}
-    </div>
+    <>
+      {data.map((node, idx) =>
+        addPaddingToParagraphs ? (
```

Comment:
curious, this if statement seems unnecessary?

Reply:
Yes.  I use Flexbox to create separation between each element.